src/js/main.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+require('../css/style.scss');
+const $ = require('jquery');
+const Handlebars = require('handlebars');
+$(($) => {
+    let slideCount = 0;
+    // TODO: Hypatia API data will be filled here, and that's it...
+    const videoCollectionDataArr = [
```

Comment:
You'll want to remove this hardcoded "demo" object from the carousel logic and pass it in instead. (Perhaps from an inline script call in the demo html file)